dist/Data-Dumper/Makefile.PL

```diff
@@ -1,10 +1,23 @@
 use strict;
 use warnings;
 use ExtUtils::MakeMaker;
+
+my @extra;
+push @extra, 'LICENSE' => 'perl_5'
+    unless $ExtUtils::MakeMaker::VERSION < 6.31;
```

Comment:
```suggestion
    if eval { ExtUtils::MakeMaker->VERSION('6.31') };
```

This will fail as a numeric comparison for underscore versions.

Reply:
This code is modeled on code in other `dist`/*/Makefile.PL` instances.  In all but two of those instances we do *not* use this `eval` syntax.  So I would rather have that change be done in a separate pull request for all of the distributions that need it -- assuming we have a consensus that it is better than the one used above.

Reply:
Yes, lots of version checks are wrong.